### DISCUSSION TITLE
Add custom giveaway bots

### DISCUSSION THREAD

**Discussion by ghostyplayz** [Vedza/NitroSniperGo#198] [Ideas]
It would be cool if, in the settings, you could add giveaway bots by their ids as I'm in a lot of servers with their own bots.

**Comment by Vedza** [comment]
It's not about the id, it's about the messages that the bot send, so I can determine if the sniper needs to react and if you won something.
The sniper try to detect giveaway only on bot account, no matter the id of the bot.
